FordLabs/PeopleMover
# 236 mysql local

## Issue
Resolves 236, making MySQL the database that gets run locally

## What was done
- [x] Set up configuration file(s) to use mysql as the database, rather than h2

## How to test
1. On startup of backend, verify that JDBC uses mysql as the source rather than H2. 
(I need to check, but I believe the pipeline will use H2.)
